RevenueCatUI/CustomerCenter/Data/Transaction.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Transaction.swift
+//
+//  Created by Facundo Menzella on 5/5/25.
+
+import Foundation
+import RevenueCat
+
+protocol Transaction {
```

Comment:
does this actually represent a transaction? if so, maybe this should have a product ID

Reply:
also, we already define `Transaction` concepts all around the SDK... if this one is just for customer center, would it make sense to namespace it under customer center?

Reply:
Yeah you're right. 
However, this is just another abstraction used inside CustomerCenter. Currently we have three:
- Transaction
- PurchaseInformation
- PurchaseInfo

I think we need to refactor this (in a separate PR to avoid making this bigger). One should be enough.